### PR TITLE
Ajd chapter2

### PR DESCRIPTION
**Pull request by alexdeas**
Chapter 2 answers

### CODE REVIEW

File: Java-Training/self_test.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  uses UTF8 instead of ascii for i18n.
+  
+3) If everything was as good as Perl this would be true. But sadly it's false.
+
+4) System.out.println( "One\nTwo\nThree" );
+
+5) 'i' doesn't appear to be declared as an int, but the bigger problem is that sum is being redeclared  
+  each loop iteration so won't preserve the sum. It's also not initialised to some value, so presumably
+  Java would complain about that too.
+
```
Comment (codavex) [inline Java-Training/self_test.txt:13]: As sum is declared inside the for loop, it's also out of scope for the println

File: Java-Training/src/EchoLocation.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/*
+ * Try this 2-1.6
+ */
+
+public class EchoLocation {
+  
+  public static void main ( String[] args ) {
+    double dist;
+    double speedOfSound = 340.29; // metres per second
```
Comment (codavex) [inline Java-Training/src/EchoLocation.java:9]: hmm, you changed from ft/s to m/s

File: Java-Training/src/SmallPrimeNumbers.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  
+  public static void main ( String[] args ) {
+    
+    for ( int number = 2 ; number <= 100 ; number++ ) {
+      int divisor_count = 0;
+      
+      for ( int divisor = 2 ; divisor <= ( number / 2 ) ; divisor++ ) {
+        if ( number % divisor == 0 ) {
+          divisor_count++;
+          // Ideally I'd break out of this loop somehow.
```
Comment (codavex) [inline Java-Training/src/SmallPrimeNumbers.java:13]: ..if only we'd read about Program Control Statements. Wonder what the next chapter is?

File: Java-Training/self_test.txt (hunk 1 above)
Comment (alexdeas) [inline Java-Training/self_test.txt:13]: I forgot to read the last line apparently.

File: Java-Training/src/EchoLocation.java (hunk 2 above)
Comment (alexdeas) [inline Java-Training/src/EchoLocation.java:9]: I deny the existence of imperial measurements.

**Pull request closed without merging**